test/integration/golden_response/search_statvar/employed_16.json

```diff
@@ -1,920 +0,0 @@
-{
-  "statVars": [
```

Comment:
I don't understand why this is dropped?

Reply:
nm, you renamed it.

Reply:
I replaced employed_16.json and employed_16_blocklist.json with accommodation_food_services.json and accommodation_food_services_blocklist.json because employed_16 doesn't have anything that would be blocklisted anymore